### DISCUSSION TITLE
Livewire pagination

### DISCUSSION THREAD

**Discussion by BNhashem16** [livewire/livewire#4881] [1. Help]
I use livewire pagination, and when i click on any page buttoms this error occured
I still debuging foe 3 hours and cant found solution!

<img width="907" alt="image" src="https://user-images.githubusercontent.com/49618985/162945692-bd99b19b-a0a8-43a2-af03-f1bf92bfe47a.png">

<img width="604" alt="image" src="https://user-images.githubusercontent.com/49618985/162945862-41d701d7-b4e0-435e-85b0-0683dd8978e5.png">

**Comment by tushargugnani** [comment]
@BNhashem16 is there any change any javascript plugin is changing the DOM on the frontend after livewire renders the results?
